chapters/es/slides/chapter2_03_word-vectors-similarity.md

```diff
@@ -98,7 +98,7 @@ print(doc.similarity(token))
 ```
 
 ```out
-0.4755507088511145
+0.13637736545255463
```

Comment:
Are these the actual results with the vectors? If so, I think we need to find a better example because we want something that actually matches the rough similarities here.

Reply:
Yup. These are what I get with the language model. I was going to find other examples but also I didn't know how much you'd like me to diverge from the English version. I'll find different ones.

Reply:
Resolved in commit 8453c5c
I also updated a few other things related to text similarities.